upload.prx.org/upload.prx.org.yml

```diff
@@ -83,3 +113,22 @@ Resources:
       Role: !GetAtt LambdaBasicExecutionIAMRole.Arn
       Runtime: nodejs4.3
       Timeout: 5
+  UploadErrorAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: "Upload lambda errors"
```

Comment:
Might be nice to include the environment type in the name so it's more obvious which is which when there are several stacks launched